mkdocs.yml

```diff
@@ -79,7 +81,7 @@ nav:
       - Nonlinear matrix approximation: examples/matrix-approximation.ipynb
     - pages/cheatsheet.md
     - User Guide:
-        - pages/user-guide/tensor-and-indices.md
+        - "Tensors and Indices": pages/user-guide/tensor-and-indices.md
```

Comment:
Do we need to add this type of descriptive string for the other pages of the user guide?

Reply:
This is the only place needed, to address a (IMHO) bug of `mkdocs` :joy: For other pages, the page's TOC title is extracted automatically from the H1 heading inside the document.